trunk/SUAVE/Methods/Performance/V_n_diagram.py

```diff
@@ -0,0 +1,411 @@
+
+    Inputs:
+    analyses.base.atmosphere               [SUAVE data type]
+    vehicle.
+      mass_properties.takeoff              [kg]
+      reference_area                       [m^2]
+      maximum_lift_coefficient             [Unitless]
+      chords.mean_aerodynamic      [meter]
+
+    Outputs:
+    airspeeds, Mach, load_factors                   [m]
+
+    Properties Used:
+    N/A
+    """        
+    
+    # ==============================================
+    # Unpack
+    # ==============================================
+    atmo = analyses.configs.base.atmosphere
+
+    for wing in vehicle.wings: 
+        reference_area  = vehicle.reference_area 
+        Cmac            = wing.chords.mean_aerodynamic
+    
+    for mission in analyses.missions:
+        Vc = mission.segments.cruise.air_speed
+
+    CLa = 5.25
+    category_tag = vehicle.category
+    minimum_lift_coefficient = -1.0
+    vehicle.minimum_lift_coefficient = minimum_lift_coefficient
+    
+    # ==============================================
+    # Computing atmospheric conditions
+    # ==============================================
+    atmo_values = atmo.compute_values(altitude,delta_ISA)
+    SL_atmo_values = atmo.compute_values(0,delta_ISA)
+    conditions  = SUAVE.Analyses.Mission.Segments.Conditions.Aerodynamics()
+
+    rho = atmo_values.density
+    sea_level_rho = SL_atmo_values.density
+    a = atmo_values.speed_of_sound
+    sea_level_gravity = atmo.planet.sea_level_gravity
+
+    # ==============================================
+    # Determining vehicle maximum lift coefficient
+    # ==============================================
+    try:   # aircraft maximum lift informed by user
+        maximum_lift_coefficient = vehicle.maximum_lift_coefficient
+    except:
+        # Using semi-empirical method for maximum lift coefficient calculation
+        from SUAVE.Methods.Aerodynamics.Fidelity_Zero.Lift import compute_max_lift_coeff
+
+        # Condition to CLmax calculation: 300KTAS @ specified altitude, ISA
+        conditions.freestream=Data()
+        conditions.freestream.density   = atmo_values.density
+        conditions.freestream.dynamic_viscosity = atmo_values.dynamic_viscosity
+        conditions.freestream.velocity  = 300. * Units.knots
+        try:
+            maximum_lift_coefficient, maximum_lift_coefficient_flaps, induced_drag_high_lift = compute_max_lift_coeff(vehicle,conditions)
+            vehicle.maximum_lift_coefficient = maximum_lift_coefficient
+        except:
+            raise ValueError, "Maximum lift coefficient calculation error. Please, check inputs"
+        
+    # ============================================================================
+    # Convert all terms to English (Used for FAR) and remove elements from arrays
+    # ============================================================================
+    altitude = altitude / Units.ft
```

Comment:
Line this up all into a block:

something          = True
something_else = False